<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customWorkersEditWidget</class>
 <widget class="QWidget" name="customWorkersEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	background-color: #dfdfdf;
	font: 600 11pt &quot;Segoe UI&quot;;
	color: #324b4c;
}

QScrollBar:vertical {
        width: 20px;
        background: none;
    }

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
        background: #dfdfdf;
    }

QPushButton {
	background-color: #c9c9c9;
	border: none;
	border-radius: 10px;
	padding: 5px;
	font-size: 12pt;
}

QPushButton:hover{
	background-color: #aeaeae;
}

#addNewTimePaperBtn {
	font-size: 11pt;
}

QDateEdit::up-button, QDateEdit::down-button{
	width: 0px;
}


QLineEdit, QDateEdit{
	border: none;
	font-size: 10pt;
	border-bottom: 1px solid #7c9399;
}

#statusBar {
	border: 1px solid #7c9399;
}
#statusBar *{
	font-size: 9pt;
}

#avrTimePerPiece, #totalSelectedPieces, #totalSelectedTime, #totalSelectedRows, #totalViewRows, #totalWorkingMins,
#piecesForProdLineEdit, #piecesProducedLineEdit {
	font-weight: 700;
}

#modelHolder QLabel, #newModelInfoHolder * , #widget_11 *{
	font-size: 10pt;
}

#calStartBtn, #calLeavingBtn {
	background-color: #dfdfdf;
	padding: 0px;
}

#optionsHolder, #optionsHolder_2 {
	border-bottom: 2px dashed #7c9399;
}

#pageTitle {
	font: 700 13pt &quot;Segoe UI&quot;;
}

#userIcon, #calendarBtn {
	background-color: #dfdfdf;
}

#refreshShiftsBtn {
	padding: 0px;
	background-color: #dfdfdf;
}

#usernameLabel {
	font-size: 11pt;
}

#workerPlaceLineEdit, #workerPositionLineEdit {
	font-weight: 700;
}

#widget_3 * {
	font-size: 10pt;
}

QComboBox{
	background-color: #dfdfdf;
	border-bottom: 2px solid #7c9399;
	border-radius: 3px;
}

QComboBox::drop-down{
	image: url(':/icons/app/assets/icons/Alt-Arrow-Down--Streamline-Solar-Broken.svg');
	width: 18px;
	height: 18px;
}

QComboBox:hover{
	color: #2b2b2c;
	font: 720 10.5pt &quot;Segoe UI&quot;;
	border-radius: 0px;
	alternate-background-color: #2c313c;
}
QComboBox::item:selected{
	color: #2b2b2c;
	font: 900 10.5pt &quot;Segoe UI&quot;;
	border-radius: 0px;
}
QComboBox::item{
	color: #324b4c;
	font: 700 10pt &quot;Segoe UI&quot;;
	height: 11px;  
	border: 4px solid transparent;
	padding: 0px 8px 0px 8px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item alignment="Qt::AlignTop">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="headerHolder" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>20</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignLeft">
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="pageTitle">
           <property name="text">
            <string>РАБОТНИЦИ</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignRight">
          <widget class="QWidget" name="userHolder" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="userIcon">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../../resources.qrc">
                <normaloff>:/icons/app/assets/icons/User--Streamline-Feather.svg</normaloff>:/icons/app/assets/icons/User--Streamline-Feather.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="usernameLabel">
              <property name="text">
               <string>admin</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft">
       <widget class="QWidget" name="workerInfoHolder" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>750</width>
          <height>200</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Нов Работник</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <property name="horizontalSpacing">
               <number>10</number>
              </property>
              <item row="0" column="2">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Група</string>
                </property>
               </widget>
              </item>
              <item row="2" column="3">
               <widget class="QLineEdit" name="worker_5_EGN"/>
              </item>
              <item row="0" column="3">
               <widget class="QComboBox" name="cehoveComboBox">
                <property name="focusPolicy">
                 <enum>Qt::ClickFocus</enum>
                </property>
               </widget>
              </item>
              <item row="2" column="4">
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Телефон</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="worker_2_sirname"/>
              </item>
              <item row="2" column="2">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>ЕГН</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="worker_4_number"/>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="worker_1_name"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Фамилия</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Номер</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Презиме</string>
                </property>
               </widget>
              </item>
              <item row="0" column="5">
               <widget class="QWidget" name="widget_5" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <property name="spacing">
                  <number>5</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QDateEdit" name="startDateEdit">
                   <property name="focusPolicy">
                    <enum>Qt::ClickFocus</enum>
                   </property>
                   <property name="displayFormat">
                    <string>dd.MM.yy 'г.'</string>
                   </property>
                  </widget>
                 </item>
                 <item alignment="Qt::AlignLeft">
                  <widget class="QPushButton" name="calStartBtn">
                   <property name="focusPolicy">
                    <enum>Qt::NoFocus</enum>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="../../../resources.qrc">
                     <normaloff>:/icons/app/assets/icons/Calendar--Streamline-Solar-Broken.svg</normaloff>:/icons/app/assets/icons/Calendar--Streamline-Solar-Broken.svg</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>20</width>
                     <height>20</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Име</string>
                </property>
               </widget>
              </item>
              <item row="1" column="4">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Дата Напускане</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="worker_3_lastname"/>
              </item>
              <item row="3" column="3">
               <widget class="QComboBox" name="paymentTypeComboBox">
                <property name="focusPolicy">
                 <enum>Qt::ClickFocus</enum>
                </property>
               </widget>
              </item>
              <item row="0" column="4">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Дата Назначение</string>
                </property>
               </widget>
              </item>
              <item row="3" column="2">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Система заплащане</string>
                </property>
               </widget>
              </item>
              <item row="2" column="5">
               <widget class="QLineEdit" name="worker_6_tel"/>
              </item>
              <item row="1" column="3">
               <widget class="QComboBox" name="positionComboBox">
                <property name="focusPolicy">
                 <enum>Qt::ClickFocus</enum>
                </property>
               </widget>
              </item>
              <item row="1" column="5">
               <widget class="QWidget" name="widget_6" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <property name="spacing">
                  <number>5</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QDateEdit" name="leaveDateEdit">
                   <property name="focusPolicy">
                    <enum>Qt::ClickFocus</enum>
                   </property>
                   <property name="displayFormat">
                    <string>dd.MM.yy 'г.'</string>
                   </property>
                  </widget>
                 </item>
                 <item alignment="Qt::AlignLeft">
                  <widget class="QPushButton" name="calLeavingBtn">
                   <property name="focusPolicy">
                    <enum>Qt::NoFocus</enum>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="../../../resources.qrc">
                     <normaloff>:/icons/app/assets/icons/Calendar--Streamline-Solar-Broken.svg</normaloff>:/icons/app/assets/icons/Calendar--Streamline-Solar-Broken.svg</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>20</width>
                     <height>20</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Длъжност</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Адрес гр.</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLineEdit" name="worker_7_townAdress"/>
              </item>
              <item row="4" column="3">
               <widget class="QLineEdit" name="worker_8_streetAdress"/>
              </item>
              <item row="4" column="2">
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>ул.</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>888</width>
        <height>534</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="workersHolder" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTableView" name="workersTableView">
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QHeaderView:section{
	font: 700 10.5pt &quot;Segoe UI&quot;;
	background-color: #dfdfdf;
	padding-left: 10px;
	padding-top:10px;
	padding-right:10px;
}
QVerticalView:section{
	min-height: 30;
}
QAbstractItemView{
	alternate-background-color: #d3d3d3;
	font: 10.5pt &quot;Segoe UI&quot;;
	selection-background-color: rgba(198, 228, 254, 45);
	selection-color: #324b4c;
}
QAbstractItemView:item{
	selection-background-color: rgba(198, 228, 254, 45);
	selection-color: #324b4c;
}
QAbstractItemView::item:selected{
	background-color: rgba(198, 228, 254, 45);
}</string>
            </property>
            <property name="cornerButtonEnabled">
             <bool>false</bool>
            </property>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
